BH/Modules/Item/ItemDisplay.cpp

```diff
@@ -793,6 +796,8 @@ bool IntegerCompare(unsigned int Lvalue,
 		return Lvalue > Rvalue;
 	case LESS_THAN:
 		return Lvalue < Rvalue;
+	case BETWEEN:
+		return (Rvalue <= Lvalue && Lvalue <= Bvalue);

```

Comment:
Is there a reason this uses `<=` and `>=` while the other operators use `<` and `>`? Might cause confusion that some operators are inclusive and others aren't

Reply:
Mostly just felt better when writing out a range to have it be inclusive rather than exclusive. I don't feel super strongly either way so I'm happy to change it if you think it's going to cause confusion.

Reply:
Sure, that's fine with me.